scripts/linters/third_party_typings_linter_test.py

```diff
@@ -0,0 +1,149 @@
+# coding: utf-8
+#
+# Copyright 2020 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Unit tests for third_party_typings_linter.py."""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import os
+
+from core.tests import test_utils
+
+import python_utils
+
+from . import linter_utils
+from . import third_party_typings_linter
+
+
+class ThirdPartyTypingsLinterTests(test_utils.GenericTestBase):
```

Comment:
@nishantwrp, Can you please check whether the 100% coverage checks for linter files is enabled?

Reply:
Yup It is